README.rst

```diff
@@ -913,3 +913,11 @@ We are planning to solve these problems in the future releases of
 ``scrapy-poet`` and ``scrapy-zyte-api``.
 
 .. _scrapy-poet provider: https://scrapy-poet.readthedocs.io/en/stable/providers.html
+
+
+Running behind a proxy
+======================
+
+If you need Zyte API requests to go through a proxy, configure the
```

Comment:
There is an opportunity to misread it, e.g. understand "Zyte API request" as a request Zyte API makes to a remote website. Would something like this be more clear?

> If you require a proxy to access Zyte API (such as corporate proxy), ...